minutes/2022-08-22.md

```diff
@@ -0,0 +1,95 @@
+# Meeting Minutes for August 22, 2022
+
+## Attendees
+
+@fviernau, @MarcelBochtler, @mnonnenmacher, @sschuberth, @tsteenbe
+
+## NOTICE file and copyright change
+
+- New files will be created with the new copyright header, see https://github.com/oss-review-toolkit/ort-governance/issues/6#issuecomment-1173522852.
+- Existing files should not be updated using the new copyright header. This will be done by Sebastian in one big change.
+
+## Marketing & Website
+
+- Thomas is working on an ORT website with @mkurzman and Kerstin Sonnenholzer.
+- Currently mainly the flow of information is being discussed.
+- The current decisions are documented in ORT's OneDrive.
+- The main goal is to have a clear communication about ORT's functionality and to help users to find information faster.
+- The first step will be, to have a public GitHub organization `README.md` within the [.github](https://github.com/oss-review-toolkit/.github) repository.
+
+## ORT Server
+
+Marcel and Martin updated the TSC about the current efforts of Bosch.IO regarding the ORT server (see: https://github.com/oss-review-toolkit/ort/issues/4688).
+
+- Experiments are being done using [Kubernetes](https://kubernetes.io) and [KEDA](https://keda.sh) as an auto-scaler.
+- The plans are to make the server cloud agnostic.
+- There are internal discussions within Bosch.IO to open source the ORT server.
```

Comment:
"open-source"

Reply:
Should we clarify that this discussion is not about the "if" (the server *will* for sure be open-sourced), but only about the "when" / "prerequisites"?

Reply:
> Should we clarify that this discussion is not about the "if" (the server will for sure be open-sourced), but only about the "when" / "prerequisites"?

No, this should not be clarified here.

Reply:
Why not? Otherwise this reads as if there is a chance that Bosch.IO's work on the server could not be open-sourced at all.

Reply:
@sschuberth I think this sentence is correct in the scope of documenting the meeting, the decision will be published separately.